README.md

```diff
@@ -54,6 +54,141 @@ async function getData() {
+| tags              | [string]                                                                                                      | no       | Tags associated with the file.                                                                                                                                                                                                                        |
+| metadata          | string                                                                                                        | no       | Metadata associated with the file.                                                                                                                                                                                                                    |
+| overwrite         | boolean                                                                                                       | no       | Overwrite flag. If set to `true`, will overwrite any file that exists with the same path, name, and type. Defaults to `false`.                                                                                                                        |
+| filenameOverride  | boolean                                                                                                       | no       | If set to `true`, will add unique characters to the name if an asset with the given name already exists. If the overwrite flag is set to `true`, preference will be given to the overwrite flag. If both are set to `false`, an error will be raised. |
+| expiry            | number                                                                                                        | no       | Expiry time in seconds for the underlying signed URL. Defaults to 3000 seconds.                                                                                                                                                                       |
+| uploadOptions     | Object                                                                                                        | no       | Additional options for fine-tuning the upload process. Default: `{ chunkSize: 10 * 1024 * 1024, maxRetries: 2, concurrency: 3, exponentialFactor: 2 }`                                                                                                |
+| chunkSize         | number                                                                                                        | no       | Size of each chunk to upload. Default is 10 megabyte. Recommended chunk size for 3g network - up to 5kb, 4g network - 500kb to 1MB, 5g network - 1MB to 2MB                                                                                           |
```

Comment:
We don't need to provide a chunk size recommendation for 3g, 4g, 5g here since this is a backend SDK.